app/views/provider/admin/account/payment_gateways/braintree_blue/edit.html.slim

```diff
@@ -60,8 +60,8 @@ p
 
 = semantic_form_for current_account.customer, as: :customer, url: hosted_success_provider_admin_account_braintree_blue_path(next_step: params[:next_step]), id: 'customer_form' do |form|
   = form.inputs "Personal Details" do
-    = form.input :first_name, required: true
-    = form.input :last_name, required: true
+    = form.input :first_name, required: true, label: 'First name'
+    = form.input :last_name, required: true, label: 'Last name'
```

Comment:
should we use some kind of i18n here? I don't know when we should and shouldn't use historically.

Reply:
Good suggestion. We should use it everywhere IMHO.